pkg/sink/cloudstorage/table_definition.go

```diff
@@ -151,20 +153,46 @@ func (t *TableCol) ToTiColumnInfo() (*timodel.ColumnInfo, error) {
 	return col, nil
 }
 
-// TableDetail is the detailed table definition used for cloud storage sink.
-type TableDetail struct {
-	Table        string     `json:"Table"`
-	Schema       string     `json:"Schema"`
-	Version      uint64     `json:"Version"`
-	Columns      []TableCol `json:"TableColumns"`
-	TotalColumns int        `json:"TableColumnsTotal"`
+// TableDefinition is the detailed table definition used for cloud storage sink.
+type TableDefinition struct {
+	Table        string             `json:"Table"`
+	Schema       string             `json:"Schema"`
+	Version      uint64             `json:"Version"`
```

Comment:
Why use two version fields?

Reply:
Version is used to track the change of the schema.json file layout. TableVersion is used to record the table version.